guide/voice/voice-connections.md

```diff
@@ -0,0 +1,113 @@
+# Voice Connections
+
+Voice connections represent a connection to a voice channel in a Guild. You can only connect to one voice channel in a Guild at a time.
+
+Voice connections will automatically try their best to re-establish their connections when they move across voice channels, or if the voice server region changes.
+
+## Cheat sheet
+
+### Creation
+
+Creating a voice connection is simple:
+
+```ts
+import { joinVoiceChannel } from '@discordjs/voice';
+
+const connection = joinVoiceChannel(myVoiceChannel);
+```
+
+If you try to call `joinVoiceChannel` on another channel in the same guild in which there is already an active voice connection, the existing voice connection switches over to the new channel.
+
+### Access
+
+You can access created connections elsewhere in your code without having to track the connections yourself. It is best practice to not track the voice connections yourself as you may forget to clean them up once they are destroyed, leading to memory leaks.
+
+```ts
+import { getVoiceConnection } from '@discordjs/voice';
+
+const connection = getVoiceConnection(myVoiceChannel.guild.id);
+```
+
+### Deletion
+
+You can destroy a voice connection when you no longer require it. This will disconnect its connection if it is still active, stop audio playing to it, and will remove it from the internal tracker for voice connections.
+
+It's important that you destroy voice connections when they are no longer required so that your bot leaves the voice channel, and to prevent memory leaks.
+
+```ts
+connection.destroy();
+```
+
+### Playing audio
+
+You can subscribe voice connections to audio players as soon as they're created. Audio players will try to stream audio to all their subscribed voice connections that are in the Ready state. Destroyed voice connections cannot be subscribed to audio players.
+
+```ts
+// Subscribe the connection to the audio player (will play audio on the voice connection)
+const subscription = connection.subscribe(audioPlayer);
+
+// subscription could be undefined if the connection is destroyed!
+if (subscription) {
+	// Unsubscribe after 5 seconds (stop playing audio on the voice connection)
+	setTimeout(() => subscription.unsubscribe(), 5_000);
+}
+```
+
+::: warning
+**Voice connections can be subscribed to one audio player at most.** If you try to subscribe to another audio player while already subscribed to one, the current audio player is unsubscribed and replaced with the new one.
```

Comment:
I'm not sure if it's intended, but this warning is duplicated with `audio_player.md#53`.

Reply:
Intended, the warnings are subtly different (audio player warns about what happens when swapping resources, voice connection warns about what happens when swapping audio players)